connect/api/connect.api

```diff
@@ -54,25 +54,29 @@ public final class com/stripe/android/connect/StripeEmbeddedComponentListener$De
 public final class com/stripe/android/connect/appearance/Appearance : android/os/Parcelable {
 	public static final field $stable I
 	public static final field CREATOR Landroid/os/Parcelable$Creator;
-	public fun <init> ()V
-	public fun <init> (Lcom/stripe/android/connect/appearance/Colors;Lcom/stripe/android/connect/appearance/CornerRadius;Lcom/stripe/android/connect/appearance/Typography;Lcom/stripe/android/connect/appearance/Button;Lcom/stripe/android/connect/appearance/Button;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;)V
-	public synthetic fun <init> (Lcom/stripe/android/connect/appearance/Colors;Lcom/stripe/android/connect/appearance/CornerRadius;Lcom/stripe/android/connect/appearance/Typography;Lcom/stripe/android/connect/appearance/Button;Lcom/stripe/android/connect/appearance/Button;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Lcom/stripe/android/connect/appearance/Colors;Lcom/stripe/android/connect/appearance/CornerRadius;Lcom/stripe/android/connect/appearance/Typography;Lcom/stripe/android/connect/appearance/Button;Lcom/stripe/android/connect/appearance/Button;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lcom/stripe/android/connect/appearance/Badge;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
 	public final fun describeContents ()I
 	public fun equals (Ljava/lang/Object;)Z
-	public final fun getBadgeDanger ()Lcom/stripe/android/connect/appearance/Badge;
-	public final fun getBadgeNeutral ()Lcom/stripe/android/connect/appearance/Badge;
-	public final fun getBadgeSuccess ()Lcom/stripe/android/connect/appearance/Badge;
-	public final fun getBadgeWarning ()Lcom/stripe/android/connect/appearance/Badge;
-	public final fun getButtonPrimary ()Lcom/stripe/android/connect/appearance/Button;
-	public final fun getButtonSecondary ()Lcom/stripe/android/connect/appearance/Button;
-	public final fun getColors ()Lcom/stripe/android/connect/appearance/Colors;
-	public final fun getCornerRadius ()Lcom/stripe/android/connect/appearance/CornerRadius;
-	public final fun getTypography ()Lcom/stripe/android/connect/appearance/Typography;
 	public fun hashCode ()I
 	public fun toString ()Ljava/lang/String;
 	public final fun writeToParcel (Landroid/os/Parcel;I)V
 }
 
+public final class com/stripe/android/connect/appearance/Appearance$Builder {
+	public static final field $stable I
+	public fun <init> ()V
+	public final fun build ()Lcom/stripe/android/connect/appearance/Appearance;
+	public final fun setBadgeDanger (Lcom/stripe/android/connect/appearance/Badge;)Lcom/stripe/android/connect/appearance/Appearance$Builder;
```

Comment:
Typically we'd name these `badgeDanger` rather than including set. But it seems like this change should go through API review in general, which would be better place to discuss.

Reply:
I'm fine with either though. Pushing up the changes shortly.